exercises/etl/test.ml

```diff
@@ -0,0 +1,35 @@
+open Core.Std
+open OUnit2
+open Etl
+
+
+let ae exp got _test_ctxt =
+  let sexpr_of l = List.sexp_of_t (Tuple2.sexp_of_t Char.sexp_of_t Int.sexp_of_t) l in
+  let printer l = Sexp.to_string (sexpr_of l) in
+  assert_equal exp got ~printer
+
+let tests =
+  ["transform one character">::
```

Comment:
The description isn't quite followed - the output map should be from lower case letters:
"... It also stores the letters in lower-case regardless of the case of the input letters ..."